### ISSUE TITLE
Problems with ranked

### ISSUE THREAD

**Issue by ursi** [djanatyn/ssbm-nix#32]
When I try to play ranked and hit start, I get
> Error
> Please update and restart Slippi Launcher to verify email and accept rules

**Comment by djanatyn** [comment]
Yeah, this had me confused as well. You'll need to use the `Slippi-Launcher-2.7.0-x86_64.AppImage`, which we don't currently have packaged.

There's a link to Slippi Launcher on https://slippi.gg/downloads:
* https://github.com/project-slippi/slippi-launcher/releases/download/v2.7.0/Slippi-Launcher-2.7.0-x86_64.AppImage

(It's the `v2.7.0` release, in the `project-slippi/slippi-launcher` repository).

You can use `appimage-run` to execute this launcher:
```
cd $PATH_TO_APPIMAGE
nix-shell -p appimage-run --run 'appimage-run Slippi-Launcher-2.7.0-x86_64.AppImage'
```

I think you need to be logged in, but after that, it will make a request to, I think, https://slippi.firebaseapp.com?, and after that you should get an email confirmation.

You'll be prompted to accept the Terms of Service. After that, don't worry about Slippi Launcher anymore - you can go back to however you were loading Ranked previously, and you should be good to go!

**Comment by djanatyn** [comment]
You'll hit some issues if you try to click any of the "open file" dialogues:
> (slippi-launcher:1454943): GLib-GIO-ERROR **: 18:14:41.763: No GSettings schemas are installed on the system


Wow, there's actually a nixpkgs issue, with someone trying to run Slippi Launcher, where they got some help back in 2020:
* https://github.com/NixOS/nixpkgs/issues/93561

This can be solved by setting the environment variable `$XDG_DATA_DIRS` to include the appropriate schemas, kinda like this:
```sh
export XDG_DATA_DIRS=${gsettings-desktop-schemas}/share/gsettings-schemas/${gsettings-desktop-schemas.name}:${gtk3}/share/gsettings-schemas/${gtk3.name}:$XDG_DATA_DIRS
```

Let me know if you hit issues with that, and I can provide more support. Alternatively, you can modify the configuration files (which are JSON), which should be located somewhere like `$HOME/.config/Slippi Launcher`.

**Comment by ursi** [comment]
I have not been playing slippi really at all since I posted this. just curious if the situation has changed.
